analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -850,6 +850,69 @@ async Task WithAsyncAwait()
     }
 }
 
+class Boxing
+{
+    void EmptyNullable()
+    {
+        int? nullable = null;
+        object implicitBoxed = nullable;
+        _ = (int)implicitBoxed;             // Compliant, true null-dereference -> no nullable value access
+        var explicitBoxed = (object)nullable;
+        _ = (int)explicitBoxed;             // Compliant
+        _ = (int)(object)(int?)(null);      // Compliant
+        _ = (int)(object)(null as int?);    // Compliant
+    }
+
+    void NullableRoundTrip()
+    {
+        int? nullable = null;
+        object implicitBoxed = nullable;
+        int? unboxedNullable = (int?)implicitBoxed;
+        _ = unboxedNullable.Value;          // Noncompliant, empty
```

Comment:
Please add a non-empty counter-example where `int? nullable = ...;` is initialized with a value.

Reply:
Added. I have also added the case where `nullable` is a method parameter, and as such, its value is unknown.

This way we should have all basic scenarios: empty, non-empty, and unknown.